### ISSUE TITLE
Libsbml Warning (SBO term consistency)

### ISSUE THREAD

**Issue by Martineya26** [sbmlteam/libsbml#356]
Hi im trying to verify a petab model with petablint validatori everything seem to work as expected except for this error in the libsbml

libSBML Warning (SBO term consistency): The value of the 'sboTerm' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, "participant physical type"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, "material entity").
Reference: L2V4 Section 5
 SBO term 'SBO:0000375' on the <species> is not in the appropriate branch.
WARNING: Generated invalid SBML model. Check messages above.
Visualization table not available. Skipping.

The sbml model im working with is the Dalla_Man2007 the libsbml installed in python is the last one 5.20 the libsbml installed for matlab is the 5.18. I dont know what is happening i thought that maybe is caused by the version of libsbml on python but i could not uninstall it. Some help please.

**Comment by fbergmann** [comment]
Hello, I assume you mean this [model from biomodels](https://www.ebi.ac.uk/biomodels/BIOMD0000000379). When I validate it in the validator, I get the same warning. If we look at the SBO terms: 

 - [375](https://www.ebi.ac.uk/ols/ontologies/sbo/terms?iri=http%3A%2F%2Fbiomodels.net%2FSBO%2FSBO_0000375)
 - [240](https://www.ebi.ac.uk/ols/ontologies/sbo/terms?iri=http%3A%2F%2Fbiomodels.net%2FSBO%2FSBO_0000240)
 - [236](https://www.ebi.ac.uk/ols/ontologies/sbo/terms?iri=http%3A%2F%2Fbiomodels.net%2FSBO%2FSBO_0000236)

what all this means, that in SBML the species should represent matter (chemicals, metabolites, what have you), while reactions should represent the processes. In this SBML file, the species however are annotated as a process. The model has further issues with parameters also coming from an unexpected branch. 

So `petablint` is correct in saying that there is a warning (or rather 9 of them) in the model file. It should not prevent you from doing anything else though. 

In conclusion, there is nothing wrong on your machine or anything to install or uninstall. You could correct the erroneous SBO annotations, or just remove them from species and the parameters. Or live with the warning.
